c2corg_ui/static/js/trackdownload.js

```diff
@@ -74,7 +74,8 @@ app.TrackDownloadController.prototype.downloadFeatures_ = function(
 /**
  * @export
  */
-app.TrackDownloadController.prototype.downloadGpx = function() {
+app.TrackDownloadController.prototype.downloadGpx = function($event) {
```

Comment:
You're right. I forgot to clean everything once it was working. I'll update the commit accordingly.